website/docs/guides/relay-resolvers/live-fields.md

```diff
@@ -0,0 +1,119 @@
+---
+id: live-fields
+title: "Live Fields"
+slug: /guides/relay-resolvers/live-fields/
+description: Modeling data that changes over time in Relay Resolvers
+---
+
+One critical difference between client state and server state is that as client state changes over time, those changes will need to be reflected in your UI. To address this, Relay Resolvers support the ability to be marked as `@live`. Live resolvers are expected to return a `LiveState` shaped object which includes methods which allow Relay to both `read()` the current value and also to `subscribe()` to changes to the value.
+
+As this value changes over time, Relay will automatically recompute any [derived fields](./derived-fields.md) that depend on this field (including transitive dependencies if the changes cascade), and also efficiently trigger the update of any components/subscribers which have read fields that updated as a result of this change.
+
+## @live
+
+To mark a resolver as live, add the `@live` docblock tag to the resolver definition. For example:
+
+```tsx
+import type { LiveState } from 'relay-runtime';
+
+/**
+ * @RelayResolver Query.counter: Int
+ * @live
+ */
+export function counter(): LiveState<number> {
+  return {
+    read: () => store.getState().counter,
+    subscribe: (cb) => {
```

Comment:
A more descriptive name (like "callback") or a typehint would do wonders here. `cb` is a bit opaque.

Reply:
Good point. Thanks for pointing that out. Will fix.